### PR TITLE
create base version

### PR DESCRIPTION
**Pull request by keyuan0701**

### CODE REVIEW

**Review by chunhua-liu — APPROVED** [review]

**Review by chunhua-liu — APPROVED** [review]

**Review by chunhua-liu — DISMISSED** [review]

**Review by wb0716 — COMMENTED** [review]
Checked!

**Review by xueli-hu — COMMENTED** [review]
good

**Review by jianhua-git — COMMENTED** [review]
checked

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "author": "Samsung Electronics Co., Ltd.",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Samsung/tools.git"
+    },
+    "dependencies": {
+        "npm-conf": "^1.1.3",
+        "got": "^11.5.2",
+        "hpagent": "^0.1.1",
+        "decompress": "4.2.1"
```
Comment (prozanne) [inline package.json]: There is no xml2js.

File: lib/tools.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+const path = require('path');
```
Comment (prozanne) [inline lib/tools.js:1]: As a being downloaded tools, giving permission is required.
(As matching each os platforms)

Comment (prozanne) [reply lib/tools.js:1]: maybe after unzip, the permission should be granted.

**Review by fjk0401 — COMMENTED** [review]
Good for me

File: package.json (hunk 1 above)
Comment (keyuan0701) [reply package.json]: done

File: lib/tools.js (hunk 2 above)
Comment (keyuan0701) [reply lib/tools.js:1]: done

**Review by prozanne — COMMENTED** [review]
LGTM

**Review by chunhua-liu — APPROVED** [review]

**Review by chunhua-liu — APPROVED** [review]

**Pull request merged**